src/main/resources/init_db.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE books (
+    id INT NOT NULL AUTO_INCREMENT,
```

Comment:
When creating a table in MySQL, use bigint column type for storing id.

see common mistakes: https://github.com/mate-academy/jv-jdbc-intro/blob/main/checklist.md
```suggestion
    id INT NOT NULL AUTO_INCREMENT,
```